src/core/connection.c

```diff
@@ -1119,6 +1119,20 @@ QuicConnReplaceRetiredCids(
     return TRUE;
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+uint64_t
+QuicGetEarliestExpirationTime(
+    _In_ const QUIC_CONNECTION* Connection
+    )
+{
+    uint64_t EarliestExpirationTime = UINT64_MAX;
+    for (QUIC_CONN_TIMER_TYPE Type = 0; Type < QUIC_CONN_TIMER_COUNT; ++Type) {
+        EarliestExpirationTime =
+            CXPLAT_MIN(EarliestExpirationTime, Connection->ExpirationTimes[Type]);
+    }
```

Comment:
```suggestion
    uint64_t EarliestExpirationTime = Connection->ExpirationTimes[0];
    for (QUIC_CONN_TIMER_TYPE Type = 1; Type < QUIC_CONN_TIMER_COUNT; ++Type) {
        if (Connection->ExpirationTimes[Type] < EarliestExpirationTime) {
            EarliestExpirationTime = Connection->ExpirationTimes[Type];
        }
    }
```
Very minor improvements this way.